<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientClass</class>
 <widget class="QWidget" name="ClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailField"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="emailError">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="pswLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pswField">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="pswError">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ristoHubName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RistoHub</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>311</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="text">
       <string>Not registered yet? Create an account!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Client.qrc"/>
 </resources>
 <connections/>
</ui>
